subprojects/core/src/main/java/org/gradle/profile/ReportGeneratingProfileListener.java

```diff
@@ -35,6 +44,14 @@ public void buildFinished(BuildProfile buildProfile) {
         ProfileReportRenderer renderer = new ProfileReportRenderer();
         File file = new File(buildDir, "reports/profile/profile-" + FILE_DATE_FORMAT.format(new Date(buildProfile.getBuildStarted())) + ".html");
         renderer.writeTo(buildProfile, file);
+        renderReportUrl(file);
+    }
+
+    private void renderReportUrl(File reportFile) {
+        StyledTextOutput textOutput = textOutputFactory.create(ReportGeneratingProfileListener.class, LogLevel.LIFECYCLE);
+        textOutput.println();
```

Comment:
I think the offender is actually the Build Scan plugin that seems to add a new line at the end. If you do not apply the Build Scan plugin then the output looks proper.